src/main/scala/mesosphere/marathon/plugin/PluginManager.scala

```diff
@@ -0,0 +1,63 @@
+package mesosphere.marathon.plugin
```

Comment:
I thought the idea is to move all new code to core (core.plugin) and align to the new structure?